src/pages/release-announce/1.3.0.md

```diff
@@ -0,0 +1,78 @@
+---
+id: rn-1.3.0
+title: Release Announcement v1.3.0
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release Announcement v1.3.0
+We are pleased to announce that the Apache YuniKorn community has voted to
+release 1.3.0. Apache YuniKorn is a standalone resource scheduler, designed
+for managing and scheduling Big Data workloads on container orchestration
+frameworks like Kubernetes for on-prem and on-cloud use cases.
+
+## Overview
+The Apache YuniKorn community has resolved 164 [JIRAs](https://issues.apache.org/jira/issues/?filter=12352777)
```

Comment:
It needs to be this filter:
https://issues.apache.org/jira/issues/?filter=12352568
This one stays and is just for this release the next release filter will be updated as soon as we release causing a breakage in the page